session.go

```diff
@@ -722,6 +794,56 @@ func (s *session) handleUnpackedPacket(packet *unpackedPacket, rcvTime time.Time
 
 	r := bytes.NewReader(packet.data)
 	var isAckEliciting bool
+	var recoverableFrame wire.Frame
```

Comment:
I think this can be moved inside `if s.mitigationOn(...) {`

Reply:
done